content/300-guides/050-database/850-using-prisma-with-planetscale.mdx

```diff
@@ -40,8 +40,8 @@ PlanetScale's branching model and design for scalability means that there are al
 
 - **Branching and deploy requests.** PlanetScale provides two types of database branches: _development branches_, which allow you to test out schema changes, and _production branches_, which are protected from direct schema changes. Instead, changes must be first created on a development branch and then deployed to production using a deploy request. Production branches are highly available and include automated daily backups. To learn more, see [How to use branches and deploy requests](#how-to-use-branches-and-deploy-requests).
 
-- **Referential actions and integrity.** To support scaling across multiple database servers, PlanetScale [does not allow the use of foreign key constraints](https://docs.planetscale.com/learn/operating-without-foreign-key-constraints), which are normally used in relational databases to enforce relationships between data in different tables, and asks users to handle this manually in their applications.   
-With Prisma you can maintain these relationships in your data and allow the use of [referential actions](/concepts/components/prisma-schema/relations/referential-actions) by using Prisma's ability to [emulate referential integrity in the Prisma client](/concepts/components/prisma-schema/relations/referential-integrity#handling-the-referential-integrity-in-prisma). For more information, see [How to enable emulation of referential integrity](#how-to-enable-emulation-of-referential-integrity).
+- **Referential actions and integrity.** To support scaling across multiple database servers, PlanetScale [does not allow the use of foreign key constraints](https://docs.planetscale.com/learn/operating-without-foreign-key-constraints), which are normally used in relational databases to enforce relationships between data in different tables, and asks users to handle this manually in their applications.  
+  With Prisma you can maintain these relationships in your data and allow the use of [referential actions](/concepts/components/prisma-schema/relations/referential-actions) by using Prisma's ability to [emulate referential integrity in the Prisma client](/concepts/components/prisma-schema/relations/referential-integrity#handling-the-referential-integrity-in-prisma). For more information, see [How to enable emulation of referential integrity](#how-to-enable-emulation-of-referential-integrity).
```

Comment:
I think was correct before to format the list item to be broken in 2 paragraphs.